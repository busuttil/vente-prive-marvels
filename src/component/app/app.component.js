import React, { Component } from "react";
import Main from "../main.component";
import { AppContainer } from "./app.styled";

class App extends Component {
  render() {
    return (
      <AppContainer>
        <Main />
      </AppContainer>
    );
  }
}

export default App;
