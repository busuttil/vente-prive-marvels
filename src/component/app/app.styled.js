import styled from "styled-components";

export const AppContainer = styled.div`
  display: flex;
  flex-direction: column;
  width: 100%;
  height: 100%;
  font-family: "Open Sans", sans-serif;
`;
