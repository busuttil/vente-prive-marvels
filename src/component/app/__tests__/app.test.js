import React from 'react';
import { shallow } from 'enzyme';
import App from '../app.component';
import { AppContainer } from '../app.styled';

describe('App', () => {
  const getWrapper = shallow(<App />);

  it('render without crashing', () => {
    expect(getWrapper).toMatchSnapshot();
  });

  it('render AppContainer class', () => {
    const wrapper = shallow(<AppContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
