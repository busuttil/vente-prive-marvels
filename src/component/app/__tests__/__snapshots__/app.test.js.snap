// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render AppContainer class 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 100%;
  height: 100%;
  font-family: 'Open Sans',sans-serif;
}

<div
  className="c0"
/>
`;

exports[`App render without crashing 1`] = `
<styled.div>
  <Main />
</styled.div>
`;
